src/Sarif.Multitool/ResultMatchSetCommand.cs

```diff
@@ -25,16 +26,16 @@ public int Run(ResultMatchSetOptions options)
             options.OutputFolderPath = options.OutputFolderPath ?? Path.Combine(options.FolderPath, "Out");
 
             ISarifLogMatcher matcher = ResultMatchingBaselinerFactory.GetDefaultResultMatchingBaseliner();
-            Formatting formatting = (options.PrettyPrint ? Formatting.Indented : Formatting.None);
+            Formatting formatting = options.PrettyPrint ? Formatting.Indented : Formatting.None;
 
-            // Remove previous results
-            if (Directory.Exists(options.OutputFolderPath))
+            // Remove previous results.
+            if (_fileSystem.DirectoryExists(options.OutputFolderPath))
```

Comment:
>DirectoryExists [](start = 28, length = 15)

New methods in `IFileSystem` improve unit testability -- may as well, since we're already injecting the interface.